src/shared/capture/capture_basler.cpp

```diff
@@ -231,9 +247,21 @@ RawImage CaptureBasler::getFrame() {
 		unsigned char* buf = (unsigned char*) malloc(capture.GetImageSize());
 		memcpy(buf, capture.GetBuffer(), capture.GetImageSize());
 		img.setData(buf);
-
 		last_buf = buf;
 
+		if(grab_result->GetPayloadType() == Pylon::PayloadType_ChunkData &&
+		GenApi::IsReadable(grab_result->ChunkTimestamp)){
+		    //const unsigned long int freq =camera->GevTimestampTickFrequency.GetValue(); // This should always be 125 MHz for Basler-ace-1300-75gc
```

Comment:
Can this differ between cameras? Then it might be reasonable to retrieve the value once on startup.

Reply:
Good suggestion, I changed this